.vib/moodle/goss/vars.yaml

```diff
@@ -0,0 +1,42 @@
+binaries:
+  - apachectl
+  - mysql
+  - php
+  - psql
+  - pg_dump
+  - pg_restore
+  - pg_dumpall
+  - render-template
+files:
+  - paths:
+      - /opt/bitnami/apache/conf/modsecurity.conf
+      - /opt/bitnami/apache/conf/unicode.mapping
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/moodle
+      - /bitnami/moodledata
+      - /opt/bitnami/moodle
+root_dir: /opt/bitnami
+sed_in_place:
+  exclude_paths:
+    # Ignore libmoodle.sh which uses sed -i
+    - /opt/bitnami/scripts/libmoodle.sh
+apache:
+  rules:
+    - RewriteRule "(\/vendor\/)" - [F]
+    - RewriteRule "(\/node_modules\/)" - [F]
+    - RewriteRule "(^|/)\.(?!well-known\/)" - [F]
+    - RewriteRule "(composer\.json)" - [F]
+    - RewriteRule "(\.lock)" - [F]
+    - RewriteRule "(\/environment.xml)" - [F]
+    - RewriteRule "(\/install.xml)" - [F]
+    - RewriteRule "(\/README)" - [F]
+    - RewriteRule "(\/readme)" - [F]
+    - RewriteRule "(\/moodle_readme)" - [F]
+    - RewriteRule "(\/upgrade\.txt)" - [F]
+    - RewriteRule "(phpunit\.xml\.dist)" - [F]
+    - RewriteRule "(\/tests\/behat\/)" - [F]
+    - RewriteRule "(\/fixtures\/)" - [F]
+    - RewriteRule "(\/package\.json)" - [F]
+    - RewriteRule "(\/Gruntfile\.js)" - [F]
```

Comment:
From the feedback I received here: https://github.com/bitnami/containers/pull/30859#discussion_r1188287732

There is no need to check every individual line.
Checking only some of them should be fine to trust that the configuration was introduced.